hushline/routes.py

```diff
@@ -231,8 +227,12 @@ def register() -> Response | str | tuple[Response | str, int]:
 
             invite_code_input = form.invite_code.data if require_invite_code else None
             if invite_code_input:
-                invite_code = InviteCode.query.filter_by(code=invite_code_input).first()
-                if not invite_code or invite_code.expiration_date < datetime.utcnow():
+                invite_code = db.session.scalars(
+                    db.select(InviteCode).filter_by(code=invite_code_input).limit(1)
+                ).first()
+                if not invite_code or invite_code.expiration_date.replace(
```

Comment:
the `datetime.replace()` call here is to convert the naive datetime to one with a timezone. We could update the schema to return non-naive datetimes, but that seemed worth deferring for now.